src/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html

```diff
@@ -60,30 +78,62 @@
 This scanner is time sensitive, and should only be used in an attempt find find stubborn and un-obvious SQL 
 injection vulnerabilities in a suspected Hypersonic database. <br>
 For this reason, the number of active scan threads should be set to the minimum when using this scanner, 
-to minimise load on the web server, application server, and database, in order to avoid false positives caused by 
+to minimize load on the web server, application server, and database, in order to avoid false positives caused by 
 load delays rather than by SQL injection delays. <br>
 The scanner tests only for time-based SQL injection vulnerabilities. 
 
-<H2>SQL Injection - MySQL, time based</H2>
+<H2>SQL Injection - MySQL (Time Based)</H2>
 
-Similar to the Hypersonic scanner, but specific to the MySQL RDBMS and SQL syntax. 
+This scanner uses MySQL-specific SQL syntax to attempt to induce time delays in the SQL statement called by the page.<br> 
+If the unmodified query is not affected by a time delay, and the modified query's delay can be controlled, 
+it is indicative of a time-based SQL Injection vulnerability in a MySQL SQL database. <br>
+This scanner is time sensitive, and should only be used in an attempt find find stubborn and un-obvious SQL 
```

Comment:
"find" is repeated (and missing "to"? "to find"?).
This typo is also present in the "Hypersonic (Time Based)" description (and other added descriptions).